tedana/workflows/tedana.py

```diff
@@ -321,10 +353,12 @@ def tedana_workflow(
     fittype="loglin",
     combmode="t2s",
     tree="kundu",
+    ica_method=DEFAULT_ICA_METHOD,
+    n_robust_runs=DEFAULT_N_ROBUST_RUNS,
     tedpca="aic",
     fixed_seed=42,
```

Comment:
This magic number also appears in multiple locations, so should be defined as a const.